pkg/client/error.go

```diff
@@ -0,0 +1,18 @@
+package nighthawk
+
+import (
+	"github.com/layer5io/meshkit/errors"
+)
+
+const (
+	ErrGRPCDialCode        = "1000"
+	ErrInvalidEndpointCode = "1001"
+)
+
+var (
+	ErrInvalidEndpoint = errors.NewDefault(ErrInvalidEndpointCode, "Endpoint not reachable")
```

Comment:
Probably cause and suggested remediation?

Reply:
Probable cause: "Endpoint is unavailable or endpoint is unreachable" - sound right?